integrations/langfuse/src/haystack_integrations/tracing/langfuse/tracer.py

```diff
@@ -148,7 +149,14 @@ def trace(self, operation_name: str, tags: Optional[Dict[str, Any]] = None) -> I
             replies = span._data.get("haystack.component.output", {}).get("replies")
             if replies:
                 meta = replies[0].meta
-                span._span.update(usage=meta.get("usage") or None, model=meta.get("model"))
+                completion_start_time = meta.get("completion_start_time")
+                if completion_start_time:
+                    completion_start_time = datetime.fromisoformat(completion_start_time)
```

Comment:
`fromisoformat` throws ValueError for irregular format provided. Let's just log it here and skip ttft in that case, wdyt?

Reply:
Sure, will do. Normally this won't be the case though.

Reply:
Yeah I agree, thanks for your understanding 👍

Reply:
Updated, and also added a new unit test for this.